manage

```diff
@@ -261,6 +259,12 @@ weblate.push.translations() {
             return 0
```

Comment:
If I'm not wrong, this return ends the subshell started in line 242 by `(` .  The shell will continue where the subshell ends, see closing  `)` below .. and `pyenv.cmd wlc unlock` will be executed .. or I'm wrong?

Reply:
How to keep `set -e` for before `pyenv.cmd wlc lock` ?

Reply:
Try this patch ...

```diff
diff --git a/manage b/manage
index 6f6f1a4f..488a93cc 100755
--- a/manage
+++ b/manage
@@ -240,9 +240,6 @@ weblate.push.translations() {
     local messages_pot diff_messages_pot last_commit_hash last_commit_detail \
           last_commit_message exitcode
     (   set -e
-        # lock change on weblate
-        pyenv.cmd wlc lock
-
         # get translations branch in git worktree (TRANSLATIONS_WORKTREE)
         weblate.translations.worktree
 
@@ -258,8 +255,18 @@ weblate.push.translations() {
                             git diff -- "searx/translations/messages.pot")
         if ! echo "$diff_messages_pot" | grep -qE "[\+\-](msgid|msgstr)"; then
             build_msg BABEL 'no changes detected, exiting'
-            return 0
+            return 1
         fi
+        return 0
+    )
+    exitcode=$?
+    if [ "$exitcode" ]; then
+       return 0
+    fi
+
+    (   set -e
+        # lock change on weblate
+        pyenv.cmd wlc lock
 
         # save messages.pot in the translations branch for later
         pushd "${TRANSLATIONS_WORKTREE}"
```